example/demo/src/inventory/ui.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  action,
+  ComponentDataState,
+  Element as JSXElement,
+  ElementType as JSXElementType,
+  ServerDataState,
+} from '@final-ui/react'
+import {
+  Button,
+  H2,
+  H4,
+  Image,
+  Paragraph,
+  ScrollView,
+  SizableText,
+  XStack,
+  YStack,
+} from '@final-ui/tamagui/server'
+
+import { UIContext } from '../types'
+import inventory, { Item } from './inventory'
+
+// Define JSX Element and ElementType
+// Need to unload `React` or remove `React.JSX.Element` and `React.JSX.ElementType` from global namespace
+// to run this
+// WIP: overwrite instead
+declare global {
```

Comment:
last tweak needed:
1. Figure out how to not load React / overwrite React namespace
2. Move this into d.ts and load automatically
These two types change what JSX returns to ComponentDataState.